app/views/practices/complexity.html.erb

```diff
@@ -237,7 +237,7 @@
                     <%= link_to practice_resources_path(@practice), class: 'usa-button usa-button--outline editor-back-to-link margin-right-1 padding-2 margin-bottom-5 text-no-underline' do %>
                         <i class="fas fa-chevron-left fa-sm text-no-underline margin-right-1"></i>Back
                     <% end %>
-                    <%= link_to practice_timeline_path(@practice), class: 'usa-button padding-2 editor-continue-link margin-right-1 margin-bottom-5' do %>
+                    <%= link_to practice_timeline_path(@practice, next: 'timeline'), class: 'usa-button padding-2 editor-continue-link continue-and-save margin-right-1 margin-bottom-5', 'data-next': 'timeline' do %>
```

Comment:
if we use the slug-hash method, we probably don't need the `data-next` attributes on all of the buttons